packages/core/lib/test.js

```diff
@@ -171,11 +171,11 @@ const Test = {
     mochaConfig.bail = config.bail;
 
     // If the command line overrides color usage, use that.
-    if (config.colors != null) mochaConfig.useColors = config.colors;
+    if (config.color != null) mochaConfig.color = config.color;
```

Comment:
```suggestion
    if (config.colors != null) mochaConfig.color = config.colors;
```

Funky, but if we don't keep this how it currently is (`config.colors`, which was a `mocha` CLI alias for `useColors`), then this will be a breaking change.

Reply:
Ok so I wrote a conditional to check both cases.